examples/nlg_server/nlg_server.py

```diff
@@ -12,6 +12,10 @@
 
 DEFAULT_SERVER_PORT = 5056
 
+DEFAULT_SANIC_WORKERS = 1
+
+app = Sanic(__name__)
```

Comment:
is there any reason you moved this here and not into the `run_server ` function?